js/coinex.js

```diff
@@ -1822,6 +1823,31 @@ module.exports = class coinex extends Exchange {
         return this.parseTrades (trades, market, since, limit);
     }
 
+    async setLeverage (leverage, symbol = undefined, params = {}) {
+        await this.loadMarkets ();
+        if (symbol === undefined) {
```

Comment:
Let's throw early before loading the markets, if the symbol argument is undefined – please move this conditional above `loadMarkets()`.